doc/user/content/manage/dbt/blue-green-deployments.md

```diff
@@ -41,8 +41,7 @@ If your dbt project includes [sinks](/manage/dbt/get-started/#sinks), you
 **must** ensure that these are created in a **dedicated schema and cluster**.
 Unlike other objects, sinks must not be recreated in the process of a blue/green
 deployment, and must instead cut over to the new definition of their upstream
-dependencies after the environment swap. The schema and cluster for your sinks should be
-included in the dbt_project definition as well.
+dependencies after the environment swap.
```

Comment:
Should we add "Do **not** include sink-dedicated schemas and clusters in the `dbt_project.yml` definition." here as well?